Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.4.3"
 [deps]
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
You can remove this too

Reply:
not sure for some reason my local julia was complaining about this if i didn't add it...not it isn't complaining when i remove it so seems fine :D

Reply:
ah i know why...because i had `import Logging` in the code before...that's why it was complaining....after that was removed, all good :)